frontend/src/models/web3/web3Gateway.js

```diff
@@ -44,7 +44,7 @@ class Web3Gateway {
         if (web3) 
             this.web3 = web3;
         else 
-            this.web3 = new Web3(Web3.givenProvider || 'ws://localhost:7545');
+            this.web3 = new Web3(Web3.givenProvider || 'wss://eth-sepolia.g.alchemy.com/v2/' + ALCHEMY_API_KEY);
```

Comment:
I assume that this is for connecting to the Testnet. Should we have an option or config in which we could specify this? It might make testing easier instead of changing the code everytime.

Reply:
This will be changed in another PR to connect to a local node in docker.